src/transport/FabricTable.h

```diff
@@ -221,7 +221,10 @@ class DLL_EXPORT FabricInfo
 
     FabricId mFabricId = 0;
 
-    static constexpr size_t KeySize();
+    // KeySize is defined in the header so we always see its definition before
+    // its uses, which is necessary for a constexpr function to actually be
+    // treated as constexpr.
+    static constexpr size_t KeySize() { return sizeof(kFabricTableKeyPrefix) + 2 * sizeof(FabricIndex); }
```

Comment:
same here: why is this not a constant? I find it obvious that `char foo[kSize]` is a constant size as a reviewer, however seeing `char foo[Size()]` leads me to think that it is dynamic.

Reply:
This could be made constant for now. The usage of key-size would change as we refactor storage of FabricInfo structure. There'll be different keys for data items that could be loaded independently.

Reply:
If we  ever put things on stack, I think it should be `kFoo`. 
if they change to require calculation, we should use platformalloc buffers.

Reply:
Will make constant.